scipy/constants/tests/test_constants.py

```diff
@@ -1,35 +1,88 @@
-from numpy.testing import assert_equal, assert_allclose
+import pytest
+
 import scipy.constants as sc
+from scipy.conftest import array_api_compatible
+from scipy._lib._array_api import xp_assert_equal, xp_assert_close
+
+
+pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_xp_backends")]
+skip_xp_backends = pytest.mark.skip_xp_backends
+
+
+class TestConvertTemperature:
```

Comment:
I already had to touch all lines here so grouping the tests does not really increase the diff